awslambda/cml/labbing_card.json.j2

```diff
@@ -91,4 +91,4 @@
     ]
 }
   
-  }]
\ No newline at end of file
+  }]
```

Comment:
how are the labbing_card.json and labbing_card.json.j2 different?  They both look like jinja